CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Contributors:
 ### Under the hood
 
 - Dispatch the core SQL statement of the new test materialization, to benefit adapter maintainers ([#3465](https://github.com/fishtown-analytics/dbt/pull/3465), [#3461](https://github.com/fishtown-analytics/dbt/pull/3461))
+- Add optional `sslcert`, `sslkey`, and `sslrootcert` arguments to the Postgres connector. ([#3473](https://github.com/fishtown-analytics/dbt/pull/3473))
```

Comment:
I'm going to slide this up to under v0.21, since we're nearing in on the final release of v0.20. I know it's a small and net-new change, but we're going to be really disciplined with the v0.21 scope and release schedule. I hope you don't mind keeping the code inside the `dbt-materialize` plugin for an extra version.

Reply:
I don't mind at all!